week-2/errors/0.js

```diff
@@ -1,9 +1,26 @@
 // Predict and explain first...
+//Ans: You're absolutely right! Calling the capitalize function as shown would result in a SyntaxError.
```

Comment:
This block of text looks like the output of an AI. Be careful to use AI only as a learning aid, not to given answers itself